armi/nuclearDataIO/cccc/cccc.py

```diff
@@ -665,30 +568,72 @@ def createRecord(self, hasRecordBoundaries=True):
         return recordClass(self._stream, hasRecordBoundaries)
 
     @classmethod
-    def readBinary(cls, fileName):
+    def readBinary(cls, fileName: str):
+        """Read data from a binary file into a data structure"""
         return cls._read(fileName, "rb")
 
     @classmethod
-    def readAscii(cls, fileName):
+    def readAscii(cls, fileName: str):
+        """Read data from an ASCII file into a data structure"""
```

Comment:
Still seems a shame that the user need to know up front whether theyre dealing with ascii or binary...

Reply:
could even just do a `read()` that `try:`s to read binary, then falls back to ascii if that fails?